src/components/left-side-navbar/LeftSideNavbar.jsx

```diff
@@ -73,42 +85,29 @@ export const LeftSideNavbar = () => {
           })}
         </List>
 
-        <Button
-          disableElevation
-          disableRipple
-          variant="contained"
-          className="btn_broadCast"
+        <CustomButton
+          onClick={() => setOpenBroadcastDialog(true)}
+          sxStyles={BroadcastBtnSx}
         >
           Broadcast
-        </Button>
-      </div>
+        </CustomButton>
+      </Box>
 
-      <ListItem
-        component="button"
-        className={classNames(styles.link, styles.accountCircle_btn)}
-        sx={AccountCircleStyles}
+      <FollowItem
+        onClick={() => setOpenLogoutDialog(true)}
+        itemComponent="button"
+        itemSxStyles={AccountCircleSx}
+        textClassName={classNames(
+          styles.listText,
+          styles.listText_active,
+          styles.accountCircle_btnText
+        )}
+        itemClassName={classNames(styles.link, styles.accountCircle_btn)}
       >
-        <ListItemAvatar sx={{ minWidth: "1rem" }}>
-          <Avatar
-            className={styles.navIcon}
-            alt="Himanshu Avatar"
-            src={HimanshuJPG}
-          />
-        </ListItemAvatar>
-        <ListItemText
-          primary="Himanshu Singh"
-          secondary="@hsnice16"
-          className={classNames(
-            styles.listText,
-            styles.listText_active,
-            styles.accountCircle_btnText
-          )}
-        />
-
         <ListItemIcon sx={{ minWidth: "1rem" }}>
           <EllipsisHorizontalIcon className={classNames(styles.ellipsisIcon)} />
         </ListItemIcon>
-      </ListItem>
-    </Grid>
+      </FollowItem>
+    </Box>
   );
 };
```

Comment:
On left navbar make the active element style width constant rather than content width, that will give good look for the page.

Reply:
Hey, thanks Deekshith for reviewing. Can you add any screenshots or elaborate on what you are saying? Because I am not getting you.

To show the active element I am just bolding the text which is increasing the font size by 1 pixel, I guess. And I think, it is looking good.

Reply:
Sorry not active style....hovering style that shrinks and expands based on content...if you make it constant width then it will be good

Reply:
On hovering, I am changing the background color and I checked it is not shrinking and expanding. Please check once more and let me know if you are seeing any different behavior.